x/auth/keeper.go

```diff
@@ -152,15 +154,15 @@ func (ak AccountKeeper) GetPubKey(ctx sdk.Context, addr sdk.AccAddress) (crypto.
 func (ak AccountKeeper) GetSequence(ctx sdk.Context, addr sdk.AccAddress) (uint64, sdk.Error) {
 	acc := ak.GetAccount(ctx, addr)
 	if acc == nil {
-		return 0, sdk.ErrUnknownAddress(addr.String())
+		return 0, sdk.ErrUnknownAddress(fmt.Sprintf("account %s does not exist", addr))
 	}
 	return acc.GetSequence(), nil
 }
 
 func (ak AccountKeeper) setSequence(ctx sdk.Context, addr sdk.AccAddress, newSequence uint64) sdk.Error {
 	acc := ak.GetAccount(ctx, addr)
 	if acc == nil {
-		return sdk.ErrUnknownAddress(addr.String())
+		return sdk.ErrUnknownAddress(fmt.Sprintf("account %s does not exist", addr))
```

Comment:
Should the `ErrUnknownAddress` take an `addr` and print out the error in the correct format?

Reply:
Sure, but that would be break the general convention of these error functions where they all just take a simple `msg string` param.

Reply:
yeah, not really crazy about that pattern. It seems to leave a lot of subtly different errors for the same issues. I think we should standardize these, but that may be out of the scope of the PR.